Services/ds/influxdb/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
             value: "true"
           volumeMounts:
             - name: {{ .Release.Name }}-data
-              mountPath: /var/lib/influxdb
+              mountPath: /var/lib/influxdb2
```

Comment:
Fixed mount path: before, the DB folder existed within the pod only since it did not match the existing mount.